config/manager/manager.yaml

```diff
@@ -75,10 +73,6 @@ metadata:
     control-plane: controller-manager
 spec:
   ports:
-    - name: yggd
```

Comment:
OCP route refers that port: https://github.com/project-flotta/flotta-operator/blob/f3906500370d89025ace44306439127cfa69bd15/config/ocp/network/route.yaml#L11